integrated_channels/sap_success_factors/models.py

```diff
@@ -179,6 +182,19 @@ def get_content_metadata_exporter(self, user):
         """
         return SapSuccessFactorsContentMetadataExporter(user, self)
 
+    def get_learner_manger(self):
+        """
+        Return a ``SapSuccessFactorsLearnerManger`` instance.
+        """
+        return SapSuccessFactorsLearnerManger(self)
+
+    def unlink_inactive_learners(self):
+        """
+        Unlink inactive SAP learners form their related enterprises
+        """
+        sap_learner_manager = self.get_learner_manger()
+        sap_learner_manager.unlink_learners()
```

Comment:
can we make it simple 
`SapSuccessFactorsLearnerManger(self).unlink_learners()`

Reply:
I did it to follow existing pattern where usually we are returning class instances.